training/long-running-queries.md

```diff
@@ -68,35 +68,37 @@ $ ./ycsb -duration 5m -tolerate-errors -concurrency 2 -rate-limit 100 'postgres:
 ### Step 2. Find running queries in the CLI
 
 1. Launch the built-in SQL client:
-    
+
     {% include copy-clipboard.html %}
-    ~~~ sql
+    ~~~ shell
     $ cockroach sql --certs-dir=certs
     ~~~
 
-2. Find the long-running queries:
+2. Find currently active queries:
 
     {% include copy-clipboard.html %}
     ~~~ sql
-    > SELECT query_id, query, start FROM [SHOW CLUSTER QUERIES];
+    > SHOW QUERIES;
     ~~~
 
-    You can find long-running queries by filtering on the `start` column, e.g.:
+    You can also filter for queries that have been running for a certain amount of time. For example, to find queries that have been running for more than 1 hour, you would run the following:
 
+    {% include copy-clipboard.html %}
     ~~~ sql
-    > SELECT query_id, query, start FROM [SHOW CLUSTER QUERIES]
-          WHERE start < (now() - INTERVAL '1 hour');
+    > SELECT * FROM [SHOW CLUSTER QUERIES]
+      WHERE start < (now() - INTERVAL '1 hour');
     ~~~
 
 3. We're done with both the load generator, so you can stop it by going to its terminals and quitting the process.
```

Comment:
Just noticing that this sentence doesn't quite make sense. @sploiselle, did we mean to say this?

> We're doing with the load generator, so you can stop it by going to its terminal and quitting the process.

Or is there a reason we're referring to "terminals" in the plural?

Reply:
edited - @sploiselle let me know if it should be changed.